public/js/controllers/charts_controller.js

```diff
@@ -16,17 +17,45 @@ const aDay = 86400 * 1000 // in milliseconds
 const aMonth = 30 // in days
 const atomsToDCR = 1e-8
 const windowScales = ['ticket-price', 'pow-difficulty']
+const lineScales = ['ticket-price']
+// index 0 represents y1 and 1 represents y2 axes.
+const yValueRanges = { 'ticket-price': [1] }
 var ticketPoolSizeTarget, premine, stakeValHeight, stakeShare
 var baseSubsidy, subsidyInterval, subsidyExponent, windowSize, avgBlockTime
 
 function usesWindowUnits (chart) {
   return windowScales.indexOf(chart) > -1
 }
 
+function isScaleDisabled (chart) {
+  return lineScales.indexOf(chart) > -1
+}
+
 function intComma (amount) {
   return amount.toLocaleString(undefined, { maximumFractionDigits: 0 })
 }
 
+function axesToRestoreYRange (chartName, origYRange, newYRange) {
```

Comment:
When a dygraph chart has a fixed y-axis `valueRange`, zooming in and later out returning to the full chart view alters the set `valueRange`.  `axesToRestoreYRange` helps to check if a specific chart has had its value range altered and recreates the required value range parameters needed to revert to the original value range.

The issue has been mentioned here https://groups.google.com/forum/#!topic/dygraphs-users/WbrphwBcRgQ but no viable solution was found. This problem is only triggered by sliding the rangeselector only.